src/main/resources/db/changelog/changes/13-create-order-table.yaml

```diff
@@ -0,0 +1,48 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-orders-table
+      author: alex_liubchenko
+      changes:
+        - createTable:
+            tableName: orders
+            columns:
```

Comment:
Just to keep consistency, if you create orders table, your file should be create-orders-table

Reply:
Agree